src/interfaces.md

```diff
@@ -32,24 +32,24 @@ fn makeKV(key: any, val: any) = new KeyVal<any, any> {
 
 will fail to compile unless both the `key` and the `val` are the same type, which may be unexpected. However, an identical interface with a different name will be matched separately, so if you have the following two otherwise identical interfaces:
 
-```rust
+```rust,ignore
 interface any {}
 interface anythingElse {}
 ```
 
 That both would match *any* type given to them, you can then use them in the `KeyVal` constructor like this:
 
-```rust
+```rust,ignore
 fn makeKv(key: any, val: anythingElse) = new KeyVal<any, anythingElse> {
   key = key
   val = val
 }
 ```
 
-Currently the interfaces have to be redeclared for this to work, but if there is demand for it, we can add an interface aliasing feature, so one could simply write:
+There is an interface aliasing feature, so one could simply write:
 
-```rust
+```rust,ignore
 interface anythingElse = any
 ```
 
-and have it do the right thing.
+and have it do the right thing. Not super useful for the `any` interface, but for complex interfaces that you want to use in functions with the variables "unlinked" from each other, it can be quite useful.
```

Comment:
an example would be cool

Reply:
Yes it would :)


Can we do that in another PR? I don't even know what kind of example we could possibly give here where it's also useful that two variables need to be worked with in the same way but don't need to be the same type that is quick to explain and relatively compact in size.

Reply:
absolutely. I remember I saw an example you had at some point over the past month, but not sure where I even saw that